### ISSUE TITLE
Pouzivanie volitelnych datovych poli

### ISSUE THREAD

**Issue by jose1711** [sk-spell/hunspell-sk#12]
premyslam.. ci uz pri upratovani flagov nebude vhodne doplnovat aj volitelne datove polia (`po`, `is` a podobne). spolu s upravou `.aff` potom bude mozne robit aj taketo veci:
```
$ hunspell  -d sk_sk -m
ženou
ženou  st:žena po:noun is:instrum

žene
žene  st:žena po:noun is:locativ
žene  st:žena po:noun is:dativ
```

**Comment by zdenop** [comment]
Ak budeš robiť úpravy aff, tak to bude mať zmysel dopĺňať resp. otestovať.

**Comment by jose1711** [comment]
ano, s upravou `aff` pocitam. je to celkom piplacka, ale vysledok podla mna celkom stoji za to. dedicated branch: https://github.com/jose1711/hunspell-sk/tree/opt_datafields, budem rad, ked mi poskytnes nejaky odozvu (zatial som tam nahodil len zenske podstatne mena).

triediaci kluc som zmenil na pad (nom, gen, dat..)
